spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```

Comment:
If we're going to use rspec, I have a couple of thoughts for that. More of an extended discussion, so we can chat on discord.